meta-flex-distro/conf/distro/flex-os.conf

```diff
@@ -294,16 +276,12 @@ SDK_NAME_PREFIX = "${DISTRO}"
 SDK_NAME = "${SDK_NAME_PREFIX}-${SDK_VERSION}-${IMAGE_BASENAME}-${MACHINE}"
 SDKPATHINSTALL = "~/${DISTRO}/sdk/${SDK_VERSION}/${IMAGE_BASENAME}-${MACHINE}"
 
-# Current multilib prefix, for non-multilib images. Ex. the lib32
-# environment-setup within a non-lib32 image SDK.
-MLPREFIX_SECONDARY = "${@d.getVar('MLPREFIX') if not d.getVar('PN').startswith(d.getVar('MLPREFIX')) else ''}"
-
 # Set the SDK title. SDK_VERSION is left out, as it's explicitly shown in the installer
-SDK_TITLE ?= "${DISTRO_NAME} ${MLPREFIX_SECONDARY}${IMAGE_BASENAME} SDK for ${MACHINE}"
-SDK_TITLE:task-populate-sdk-ext:flex-os = "${DISTRO_NAME} ${MLPREFIX_SECONDARY}${IMAGE_BASENAME} Extensible SDK for ${MACHINE}"
+SDK_TITLE ?= "${DISTRO_NAME} ${IMAGE_BASENAME} SDK for ${MACHINE}"
+SDK_TITLE:task-populate-sdk-ext:flex-os = "${DISTRO_NAME} ${IMAGE_BASENAME} Extensible SDK for ${MACHINE}"
```

Comment:
Kindly build SDK and see if there is any impact on CB integrations. does CB picks it up successfully?

Reply:
This MLPREFIX_SECONDARY is empty variable in our case, and it was effective only for multilib feature to add the MLPREFIX.